packages/oui-navbar/_mixins.less

```diff
@@ -67,6 +67,15 @@
         color: inherit;
       }
     }
+
+    // Header/Title
+    &__title {
+      display: inline-block;
+      margin-left: rem-calc(15);
+      font-size: rem-calc(28);
+      font-weight: 300;
```

Comment:
No variables for `15`, `28`, `300` and `#fff` ?

Reply:
Added